scripts/svm/executeRebalanceToHubPool.ts

```diff
@@ -0,0 +1,482 @@
+/**
+ * Script: Execute USDC Rebalance to Hub Pool
+ *
+ * This script executes a previously proposed root bundle on the Hub Pool
+ * to rebalance USDC from the Solana Spoke Pool to the Ethereum Hub Pool.
+ * It handles CCTP attestations, relayer refund leaf execution, and prepares
+ * pending liabilities for bridging back to the Hub Pool.
```

Comment:
just an irritating nit that like with the other scripts we have 120 chars to work with so we don't need to wrap shorter than 120.

Reply:
Fixed!